webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/SnapshotState.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.backup.repository.elasticsearch;
+
+public enum SnapshotState {
+  FAILED("FAILED"),
+  PARTIAL("PARTIAL"),
+  IN_PROGRESS("IN_PROGRESS"),
+  SUCCESS("SUCCESS");
+  private final String state;
+
+  SnapshotState(final String state) {
+    this.state = state;
+  }
+
+  @Override
+  public String toString() {
+    return state;
```

Comment:
:question: Why overload `toString` if we always return the name, and not use `SnapshotState.FAILED.name()`?

Reply:
It's probably not needed :+1: